storage/entities/migration.js

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const { defaults, pick } = require('lodash');
+const { NonSupportedOperationError } = require('../errors');
+const filterType = require('../utils/filter_types');
+const BaseEntity = require('./base_entity');
+
+const defaultCreateValues = {};
+
+/**
+ * Migration
+ * @typedef {Object} Migration
+ * @property {string} id
+ * @property {string} name
+ */
+
+/**
+ * Migration Filters
+ * @typedef {Object} filters.Migration
+ * @property {number} [id]
+ * @property {number} [id_eql]
+ * @property {number} [id_ne]
+ * @property {number} [id_in]
+ * @property {number} [id_like]
```

Comment:
`id` filter type is `text` meaning this should be `{string}`

Reply:
Done. Resolving.